tools/build/premake

```diff
@@ -54,15 +56,15 @@ setup_premake_path_override()
 def main():
   # First try the freshly-built premake.
   premake5_bin = os.path.join(premake_path, 'bin', 'release', 'premake5')
-  if not has_bin(premake5_bin):
```

Comment:
The has_bin function searches for the binary globally in the PATH. The code here builds a path to the expected premake location `xenia/third_party/premake-core/bin/release/premake5` which is not in the user's PATH. This change just checks to see if the binary is available at that location instead.